dcrtimed/backend/filesystem/filesystem.go

```diff
@@ -72,7 +71,9 @@ type FileSystem struct {
 	duration time.Duration // How often we combine digests
 	commit   uint          // Current version, incremented during flush
 
-	enableCollections bool // Set to true to enable collection query
+	enableCollections bool  // Set to true to enable collection query
+	confirmations     int32 // Number of confirmations to return timestamp proof
+	maxDigestsNumber  int32 // Number of confirmations to return timestamp proof
```

Comment:
Keep them short; long names obscure what the code does. Avoid redundant names, given their context.
https://go.dev/talks/2014/names.slide#6

maxDigestsNumber -> maxDigests
We already know maxDigests is a number because it's an int32, you don't need to include it in the name.